raiden/api/webui/static/endpointConfig.js

```diff
@@ -1,6 +1,6 @@
-const backendUrl='http://localhost:5002';
-const nodeAddress = '0x62Aa1173561C4E92876B3D073B9767A2d07496f8';
-const rnsDomain = null;
+const backendUrl='http://localhost:5002'; 
```

Comment:
we don't need to make changes to this file, right?

Reply:
No, we don't. Accidental commit

Reply:
Remove this change please